server/use-cases/implementation/update-session/UpdateSessionInteractor.ts

```diff
@@ -0,0 +1,32 @@
+import Session from "../../../entities/session/Session";
+import SessionOutputData from "../../../output-data/SessionOutputData";
+import SessionGateway from "../../../data-gateway/SessionGateway";
+import { GameState } from "../../../entities/game-state/GameState";
+import SessionInputData from "../../../input-data/SessionInputData";
+import UpdateSessionUseCase from "../../input-boundary-models/UpdateSessionUseCase";
+
+export default class UpdateSessionInteractor implements UpdateSessionUseCase {
+  private sessionGateway: SessionGateway;
+
+  constructor(sessionGateway: SessionGateway) {
+    this.sessionGateway = sessionGateway;
+  }
+
+  update(data: SessionInputData): SessionOutputData {
+    const session = this.sessionGateway.findById(data.getSessionId());
```

Comment:
Čia turbūt jei nerado, reiktų throw error. Sumažėtų tikrinimų nereikalingų ir metodo output būtų logiškas(dabar outputina game be ID :D )